src/main/scala/ch/epfl/bluebrain/nexus/sbt/nexus/ReleasePlugin.scala

```diff
@@ -42,9 +43,9 @@ object ReleasePlugin extends AutoPlugin {
            |    "issuesurl": "${homepage.value.map(_.toString + "/issues").getOrElse("undefined")}"
            |
            |  },
-           |"license": "Apache License 2.0",
+           |"license": "${licenses.value.map(_._1).mkString(",")}",
            |"author": "Blue Brain Nexus Team",
-           |"contributors": ["Blue Brain Nexus Team"]
+           |"contributors": [${contributors.map(c => s"\042$c\042").mkString(",")}]
```

Comment:
please use triple quotation instead of unicode:

`c => s""""$c""""` or even better in this particular case:

`${contributors.mkString("\"", "\",\"", "\"")}`